ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py

```diff
@@ -192,6 +192,35 @@ def test_paddle_asin(
     )
 
 
+# log_softmax
+@handle_frontend_test(
+    fn_tree="paddle.tensor.math.log_softmax",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+    ),
+)
+def test_paddle_log_softmax(
+    *,
+    dtype_and_x,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+    fn_tree,
+    frontend,
+    test_flags,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        x=x[0],
+    )
+
```

Comment:
The test is passing! 
Only thing to fix is a lint formatting error. Please have a look at `lint / Check formatting / check-formatting (pull_request)` in the CI checks at the bottom of the PR.